pkg/request/config.go

```diff
@@ -178,3 +182,20 @@ type CreateGuildXPRole struct {
 	RoleID  string `json:"role_id" binding:"required"`
 	XP      int    `json:"xp"`
 }
+
+type UpsertGuildConfigTipRangeRequest struct {
+	GuildID string   `json:"guild_id"`
+	MinRain *float64 `json:"min_rain"`
+	MaxTip  *float64 `json:"max_tip"`
+}
+
+func (r *UpsertGuildConfigTipRangeRequest) Validate() error {
```

Comment:
what if 
1. min == max
2. min > max
3. min default should always = 0

Reply:
ok, just don't want to confuse, let me fix it